dss-api

```diff
@@ -1,9 +1,31 @@
 #!/usr/bin/env python
 
+"""
+Entry point for starting a local test HCA DSS API server.
+"""
+
+import os
+import sys
 import logging
+import argparse
+
+from chalice.cli import CLIFactory, run_local_server
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument("--port", type=int, default=5000)
+parser.add_argument("--no-debug", dest="debug", action="store_false")
```

Comment:
i actually don't know what the difference is between debug vs log level, and it's quite possible the next person who fires this up also does not know.